pkg/oc/admin/policy/modify_roles.go

```diff
@@ -568,9 +568,7 @@ func printSuccessForCommand(role string, didAdd bool, targetName string, targets
 	if isNamespaced {
 		clusterScope = ""
 	}
-	if len(targets) > 1 {
-		targetName = targetName + "s"
```

Comment:
Something is not right.  Remove the arg if it isn't used, but it seems more likely that the block is wrong.